cp-url.go

```diff
@@ -31,6 +31,15 @@ type copyURLs struct {
 	Error         *probe.Error `json:"-"`
 }
 
+func (m *copyURLs) Equal(o interface{}) bool {
```

Comment:
Because this was already in the code, I thought it was done for a reason. I've done quite some refactoring already, but wanted to keep as close as the original code as possible.